src/test/java/com/google/cloud/spanner/jdbc/it/ITJdbcPgNumericTest.java

```diff
@@ -243,7 +243,9 @@ public void testPreparedStatement() throws Exception {
       preparedStatement.setObject(15, new BigDecimal("1"));
       preparedStatement.setObject(16, Value.pgNumeric("1"));
 
+      // TODO: Change to setFloat(..) when float32 is supported.
       preparedStatement.setFloat(17, Float.NaN);
```

Comment:
Should this be commented out? (17 is being set twice)

Reply:
Yes, to be consistent it should. (It doesn't really matter, as setting 17 twice means that the second call will overwrite the first, which means that it is being set to a double)